w3af/plugins/audit/xss.py

```diff
@@ -107,8 +112,9 @@ def _check_xss_in_parameter(self, mutant, debugging_id):
         """
         Tries to identify (persistent) XSS in one parameter.
         """
-        if not self._identify_trivial_xss(mutant):
-            self._search_xss(mutant, debugging_id)
+
+        if self._identify_trivial_xss(mutant) == False:
```

Comment:
Why break something that was ok? :-)

https://www.python.org/dev/peps/pep-0008/ says:

> Don't compare boolean values to True or False using ==.
> Yes:   if greeting:
> No:    if greeting == True:
> Worse: if greeting is True: